src/controllers/dynakube/dtversion/docker.go

```diff
@@ -13,7 +13,12 @@ import (
 )
 
 // VersionLabel is the name of the label used on ActiveGate-provided images.
-const VersionLabel = "com.dynatrace.build-version"
+const (
+	VersionLabel                   = "com.dynatrace.build-version"
+	TmpCAPath                      = "/tmp/dynatrace-operator"
+	TmpCAName                      = "dynatraceCustomCA.crt"
+	CustomCertificatesConfigMapKey = "certs"
```

Comment:
wouldn't `CustomCertificatesConfigMapKey ` fit better in the `dtclient` package ?

Reply:
moved